### ISSUE TITLE
Crash after right click

### ISSUE THREAD

**Issue by iSpeeX** [files-community/Files#4084]
Hello !
My bug is really annoying as the right click is simply never working. Each time I use it, Files crash and I need to close it down to be able to use it again. I tried to change some settings but still the same.
Folders and files are concerned and I tried dozens of times before posting here.
Can anyone help me please ?

Thank you !

**System Information**
Windows 10 Pro
Files version : 1.3.0.0

**Log**
```
2021-03-15 20:57:18.8087|INFO|FilesFullTrust.Program|Argument: DetectQuickLook|
2021-03-15 20:57:18.8177|INFO|FilesFullTrust.QuickLook|QuickLook detected: True|
2021-03-15 20:57:19.6465|INFO|FilesFullTrust.Program|Argument: GetOneDriveAccounts|
2021-03-15 20:57:19.6585|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:25.4581|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:25.4780|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:25.4900|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:25.5050|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:33.4265|INFO|FilesFullTrust.Program|Argument: Terminate|
2021-03-15 20:57:37.5695|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:37.6423|INFO|FilesFullTrust.Program|Argument: GetOneDriveAccounts|
2021-03-15 20:57:37.6913|INFO|FilesFullTrust.Program|Argument: DetectQuickLook|
2021-03-15 20:57:37.6913|INFO|FilesFullTrust.QuickLook|QuickLook detected: True|
2021-03-15 20:57:37.8399|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:38.0423|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:38.2445|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:38.4482|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:38.6515|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:38.8551|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:39.0585|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:39.2629|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:39.3986|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:39.4105|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:39.4200|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:39.4200|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:39.5497|INFO|FilesFullTrust.Win32API|Impossible d'utiliser un objet COM qui a été séparé de son RCW sous-jacent. L'objet COM a été libéré alors qu'il était toujours utilisé sur un autre thread.|System.Runtime.InteropServices.InvalidComObjectException: Impossible d'utiliser un objet COM qui a été séparé de son RCW sous-jacent. L'objet COM a été libéré alors qu'il était toujours utilisé sur un autre thread.
   à System.StubHelpers.InterfaceMarshaler.ConvertToManaged(IntPtr pUnk, IntPtr itfMT, IntPtr classMT, Int32 flags)
   à Vanara.PInvoke.Shell32.SHGetImageList(SHIL iImageList, Guid& riid, IImageList& ppv)
   à FilesFullTrust.Win32API.GetFileIconAndOverlay(String path, Int32 thumbnailSize)
   à FilesFullTrust.Win32API.<>c__DisplayClass0_0`1.<StartSTATask>b__0()
2021-03-15 20:57:39.5497|INFO|FilesFullTrust.Win32API|Impossible d'utiliser un objet COM qui a été séparé de son RCW sous-jacent.|System.Runtime.InteropServices.InvalidComObjectException: Impossible d'utiliser un objet COM qui a été séparé de son RCW sous-jacent.
   à System.StubHelpers.InterfaceMarshaler.ConvertToManaged(IntPtr pUnk, IntPtr itfMT, IntPtr classMT, Int32 flags)
   à Vanara.PInvoke.Shell32.SHGetImageList(SHIL iImageList, Guid& riid, IImageList& ppv)
   à FilesFullTrust.Win32API.GetFileIconAndOverlay(String path, Int32 thumbnailSize)
   à FilesFullTrust.Win32API.<>c__DisplayClass0_0`1.<StartSTATask>b__0()
2021-03-15 20:57:39.5636|INFO|FilesFullTrust.Program|Argument: GetSharePointSyncLocationsFromOneDrive|
2021-03-15 20:57:40.0364|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-03-15 20:57:40.9148|INFO|FilesFullTrust.Program|Argument: LoadContextMenu|
```

Best regards.

**Comment by gave92** [comment]
Doe the Files window "freeze" or does it close on its own?

**Comment by iSpeeX** [comment]
Hello,

The Files windows "freeze" as you said, no force-close at all. The only thing "working" is the hover effect on the buttons, but that's it and I can't do anything to make it right except close it myself.

**Comment by Mikami382** [comment]
I'm exactly having this issue. I just heard about this and fresh installed it cause it looked nice, but it is still buggy I can't even right click any files/folder.

**Comment by iSpeeX** [comment]
I did the exact same thing. Still looking forward to use on a daily basis but without the right click it's not even possible.
Please provide your system infos and the logs files at least, it could help them to handle it and found it what's going on.

**Comment by gave92** [comment]
Thanks @iSpeeX I can the confirm that this issue has been fixed in the beta version (1.4.1) and the fix will ship to everyone in the next public version.

**Comment by iSpeeX** [comment]
Great ! Is there any way to enable "beta" program from Files ?
